src/common-components/tests/FormField.test.jsx

```diff
@@ -60,84 +66,118 @@ describe('PasswordField', () => {
+    const { getByLabelText } = render(reduxWrapper(<IntlPasswordField {...props} />));
+    const passwordInput = getByLabelText('Password');
 
-    passwordField.find('button[aria-label="Show password"]').simulate('click');
-    expect(passwordField.find('input').prop('type')).toEqual('text');
+    const showPasswordButton = getByLabelText('Show password');
+    fireEvent.click(showPasswordButton);
+    expect(passwordInput.type).toBe('text');
 
-    passwordField.find('button[aria-label="Hide password"]').simulate('click');
-    expect(passwordField.find('input').prop('type')).toEqual('password');
+    const hidePasswordButton = getByLabelText('Hide password');
+    fireEvent.click(hidePasswordButton);
+    expect(passwordInput.type).toBe('password');
   });
 
   it('should show password requirement tooltip on focus', async () => {
-    const passwordField = mount(reduxWrapper(<IntlPasswordField {...props} />));
+    const { getByLabelText } = render(reduxWrapper(<IntlPasswordField {...props} />));
+    const passwordInput = getByLabelText('Password');
     jest.useFakeTimers();
     await act(async () => {
-      passwordField.find('input').simulate('focus');
+      fireEvent.focus(passwordInput);
       jest.runAllTimers();
     });
-    passwordField.update();
+    const passwordRequirementTooltip = document.querySelector('#password-requirement-left');
 
-    expect(passwordField.find('#password-requirement-left').exists()).toBeTruthy();
+    expect(passwordRequirementTooltip).toBeTruthy();
   });
 
   it('should show all password requirement checks as failed', async () => {
     props = {
       ...props,
       value: '',
     };
-
+    const { getByLabelText } = render(reduxWrapper(<IntlPasswordField {...props} />));
+    const passwordInput = getByLabelText('Password');
     jest.useFakeTimers();
-    const passwordField = mount(reduxWrapper(<IntlPasswordField {...props} />));
     await act(async () => {
-      passwordField.find('input').simulate('focus');
+      fireEvent.focus(passwordInput);
       jest.runAllTimers();
     });
-    passwordField.update();
 
-    expect(passwordField.find('#letter-check span').prop('className')).toEqual('pgn__icon mr-1 text-light-700');
-    expect(passwordField.find('#number-check span').prop('className')).toEqual('pgn__icon mr-1 text-light-700');
-    expect(passwordField.find('#characters-check span').prop('className')).toEqual('pgn__icon mr-1 text-light-700');
+    const letterCheckIcon = document.querySelector('#letter-check span');
+    const numberCheckIcon = document.querySelector('#number-check span');
+    const charactersCheckIcon = document.querySelector('#characters-check span');
+
+    expect(letterCheckIcon).toBeTruthy();
+    expect(letterCheckIcon.className).toContain('pgn__icon');
+    expect(letterCheckIcon.className).toContain('mr-1');
+    expect(letterCheckIcon.className).toContain('text-light-700');
```

Comment:
can we do this?

```suggestion
    expect(letterCheckIcon).toBeTruthy();
    expect(letterCheckIcon.className).toContain('pgn__icon mr-1 text-light-700');
```

Reply:
Yeah sure. I have changed this. Thank you